llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -1112,6 +1112,19 @@ struct SVEIntrinsicInfo {
     return *this;
   }
 
+  bool hasMatchingIROpode() const { return IROpcode != 0; }
```

Comment:
There's an existing comment above that hopefully gives enough information but please shout if you feel something more bespoke is required.